### ISSUE TITLE
For property model

### ISSUE THREAD

**Issue by mkezmkez** [mkezmkez/property-manage#22]
for the properties model 

we need
TITLE
STREET NAME
HOUSE NUMBER
NEIGHBORHOOD --- COLONIA
CITY
MUNICIPALITY ---DELEGACION 
LATITUDE / LONGITUDE FROM GOOGLE MAPS
NUMBER OF BEDROOMS
BATHROOMS
AVAILABLE FROM DATE
PRICE
LEASE TYPE / regular or premium
SECURITY DEPOSIT
MINIMUN LEASE
PROPERTY TYPE
VERIFIED BOOLEAN
MAIN DESCRIPTION
GATED COMMUNITY / BUILDING PERKS
PETS
SPECIFIC RULES
SERVICES INCLUDED
FURNISHED BOOLEAN
FURNISHED-DESCRIPTION TEXT
WATER BOILER, AIR CONDITIONING, STOVE, HEATER, OVEN, 
PARKING INTEGER
CLOSETO
TRANSPORTATION
BIG AVENUES AROUND
ABOUT CITY AREA
HASCONTRACT
VISIT COUNT

**Comment by mkezmkez** [comment]
property type would be normal / premium
WATER BOILER, AIR CONDITIONING, STOVE, HEATER, OVEN could be all in a string or array.

**Comment by uryelah** [comment]
>WATER BOILER, AIR CONDITIONING, STOVE, HEATER, OVEN,
The above should be included in perks or services

**Comment by uryelah** [comment]
> FURNISHED-DESCRIPTION TEXT

The above can be in the normal description

**Comment by uryelah** [comment]
These three can all be the same right? Close amenities or something: 
>CLOSETO
>TRANSPORTATION
>BIG AVENUES AROUND

**Comment by uryelah** [comment]
> ABOUT CITY AREA

We can just pull this from google or wikipedia, not a unique information

**Comment by uryelah** [comment]
I'm worried this table is getting too inflated, maybe we could create a propertyDetails table?

**Comment by mkezmkez** [comment]
I agree greatly, let's do it that way, this approach would be cheaper in terms of memory when doing queries. We only need to get those details when viewing the property and not when indexing.

I put all of those fields separately to encourage landlords to fill out more information than other sites do. Those would be specific questions for them to fill out and display as such on the show page.

**Comment by mkezmkez** [comment]
> > WATER BOILER, AIR CONDITIONING, STOVE, HEATER, OVEN,
> > The above should be included in perks or services

I was thinking we could do something like this with those as hashes with booleans so we can add 
(didn't test, just writing pseudo pseudo code :)
<h1> Included basics </h1>
<ul>
<% @prop.basic_furnish.each do |furnish| %>
<li> <%= furnish.key %> : <%= furnish.value? ? "SI" : "NO" %></li>
<% end %>
</ul>

**Comment by uryelah** [comment]
That seems complex, are you thinking on storing that in the JSON format perhaps? Or just an array of objects?

**Comment by mkezmkez** [comment]
I was thinking an array of objects, IDK what you think
